### DISCUSSION TITLE
Thank you for this!

### DISCUSSION THREAD

**Discussion by darkhog** [Sehelitar/Kick.bot#7] [General]
Now I can finally enjoy songrequest on Kick (with few additional plugins).

**Comment by Sehelitar** [comment]
Hey! 👋
I'm glad you like it. I hope everything works well and thank you for your feedback 😍

**Reply by darkhog** [reply]
Works great!

**Comment by Chico3tm** [comment]
May I ask how you did you get song request to work??

**Reply by Chico3tm** [reply]
When I try to set up next song it works with the test trigger but not with the command in kick chat that I created.

**Reply by darkhog** [reply]
It's a hodgepodge on my end right now, one huge kludge. So I don't want to share any info until I get something better. For example, I don't have any queue system set up and people need to wait until one song ends before they request another. It's based on this thing: https://extensions.streamer.bot/docs?topic=115 so it only support youtube links and as I've said, no queue so people need to wait for their turn.

**Reply by Chico3tm** [reply]
oh ok thanks that helped. Yea so I have a next song command for my cider music player. So chat can request the next song but its set for 1 request per hour for the user that use the command. but adding song request is still a great add for me.
